content/doc/book/system-administration/reverse-proxy-configuration-nginx.adoc

```diff
@@ -128,7 +129,15 @@ the Jenkins group:
 
 [source,sh]
 ----
-useradd -aG jenkins nginx
+usermod -aG jenkins nginx
+----
+
+If the last command failed due that `nginx` user was not successfully found in the system,
+then you can try adding the `www-data` user to the Jenkins group:
```

Comment:
Minor (optional) phrasing change
```suggestion
If the last command failed because the `nginx` user is not defined in the system,
then you can try adding the `www-data` user to the Jenkins group:
```